tests/conftest.py

```diff
@@ -31,6 +32,11 @@ def pytest_collection_modifyitems(config, items):
             item.add_marker(skip_slow)
 
 
+@pytest.fixture(scope="session")
```

Comment:
I think this will save the output diagnostic file to this directory but it doesn't look like you needed to ignore that?

Reply:
I don't understand this comment